src/main/java/nl/tudelft/ti2206/group9/gui/Style.java

```diff
@@ -27,12 +28,80 @@
 @SuppressWarnings("restriction")
 public final class Style {
 
+    /** Hide public constructor. */
+    private Style() { }
+
+    /** BRICK material for walls, brick wall texture. */
+    public static final PhongMaterial BRICK = new PhongMaterial();
+
+    /** MOSS material for walls, mossy brick wall texture. */
+    public static final PhongMaterial MOSS = new PhongMaterial();
+
+    /** CRACK material used for walls, cracked brick wall texture. */
+    public static final PhongMaterial CRACK = new PhongMaterial();
+
+    /** FLOOR material used for floors, mossy cobblestone floor texture. */
+    public static final PhongMaterial FLOOR = new PhongMaterial();
+
+    /** COIN material used for coins, question mark box coin texture. */
+    public static final PhongMaterial COIN = new PhongMaterial();
+
+    /** WOOD material used for logs, wooden planks obstacle texture. */
+    public static final PhongMaterial WOOD = new PhongMaterial();
+
+    /** PILLAR material used for pillars, stack of cracked stone bricks texture. */
+    public static final PhongMaterial PILLAR = new PhongMaterial();
+
+    /** FENCE material used for fences, mossy brick stone texture.*/
+    public static final PhongMaterial FENCE = new PhongMaterial();
+
+    /** PLAYER material used for the player */
+    public static final PhongMaterial PLAYER = new PhongMaterial();
+
+    /** Scale for button hover (in pause menu) */
     private static final double BUTTON_HOVER_SCALE = 1.2;
 
-	/**
-     * Private empty constructor.
+     /**
+     * Method is called once to load all textures.
+     * They are loaded into Phongmaterials.
      */
-	private Style() { }
+    public static void loadTextures() {
```

Comment:
Method is too long. I suggest putting `"nl/tudelft/ti2206/group9/gui/" in a local variable, then you don't have to split the lines.